src/internal.rs

```diff
@@ -14,10 +14,14 @@ use std::time;
 
 use exec::CommandTemplate;
 use lscolors::LsColors;
-use regex::RegexSet;
+use regex::{Regex, RegexSet};
 use regex_syntax::hir::Hir;
 use regex_syntax::Parser;
 
+lazy_static! {
+    static ref SIZE_CAPTURES: Regex = { Regex::new(r"(?i)^([+-])(\d+)([bkmgt]i?)b?$").unwrap() };
```

Comment:
This still allows for `bb` and `bib`, right? The `b` in the first character clause should be removed, I think. In order to make the whole unit optional, just add `?` after the parens:

`([kmgt]i?)?b?`